client/post-editor/editor-drawer/index.jsx

```diff
@@ -62,19 +61,17 @@ const EditorDrawer = React.createClass( {
 	},
 
 	currentPostTypeSupports: function( feature ) {
-		if ( ! this.props.site ) {
+		const { site, postTypes, type } = this.props;
+		if ( ! site ) {
 			return false;
 		}
 
-		const types = this.props.postTypes.get( this.props.site.ID );
-		const currentType = find( types, { name: this.props.type } );
-
-		// assume positively if we can't determine support
-		if ( ! currentType || ! currentType.supports ) {
+		// Default to true until post types are known
+		if ( ! postTypes ) {
```

Comment:
Ah I see it is returning `true` - and that was the prior logic.  Worst case I guess it could cause some flicker of a sidebar module.